test/data/iris_decimal_marker_as_comma.txt

```diff
@@ -0,0 +1,151 @@
+#Label	Sepal length	Sepal width	Petal length	Petal width
+0	5,1	3,5	1,4	0,2
+0	4,9	3,0	1,4	0,2
```

Comment:
_(comment appended to code to allow for threaded discussion)_

I wonder what's the behavior (without this PR) for this case (asume that tab is the separator, and that both col1 and col2 are floats):
```
col1     col2
10.4    10,400.5
12.5    12,000
```

I'm guessing that we're actually able to handle `10,400.5` and actually parse it as `10400.5`. If we already handle this case, then further inspection would required on the DoubleParser to see where else are we using ',' as thousands separator... and then change your PR accordingly so that when using `,` as decimal separator then we use `.` as thousands separator. Adding a test for this would also be useful.

Reply:
Hey @antoniovs1029 , I tested two very small datasets, and called `.Preview()` without and with my changes.
Dataset 1 (decimal marker is `'.'`, thousandth marker is `','`):
> #Label	Sepal length	Sepal width	Petal length	Petal width
> 0	50,000.1	3.5	1.4	0.2
> 0	4.9	3.0	1.4	0.2

Without changes on Dataset 1:
> Key: 0 Values: NaN 3.5 1.4 0.2
> Key: 0 Values: 4.9 3.0 1.4 0.2

With changes (`DecimalMarker = '.'`) on Dataset 1:
> Key: 0 Values: NaN 3.5 1.4 0.2
> Key: 0 Values: 4.9 3.0 1.4 0.2


Dataset 2 (decimal marker is `','`, thousandth marker is `'.'`):
> #Label	Sepal length	Sepal width	Petal length	Petal width
> 0	50.000,1	3,5	1,4	0,2
> 0	4,9	3,0	1,4	0,2

Without changes  on Dataset 2:
> Key: 0 Values: NaN NaN NaN NaN 
> Key: 0 Values: NaN NaN NaN NaN 

With changes (`DecimalMarker = ','`)  on Dataset 2:
> Key: 0 Values: NaN 3.5 1.4 0.2
> Key: 0 Values: 4.9 3.0 1.4 0.2

As you can see, we're not able to handle the thousandth marker currently in ML .NET. This can be a future feature addition. Also notice in the results of Dataset 2 with changes, the decimal marker is `'.'`. This is because my system's culture is in `en-US`. Were it another culture where the decimal marker is `','`, then it would print out with the decimal markers being `','`.

Reply:
Ok,since we originally didn't support parsin "50,001.5" to "50001.5", then my comment can be dismissed and no further changes would be required on your PR on this regard 👍